### PR TITLE
Showcard bugs

### PR DESCRIPTION
**Pull request by jmenghan**
## Related Issue
Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

## Description
Create a [descriptive title and provide all of the relevant details in the description](https://chris.beams.io/posts/git-commit/). 
This information lives with the code and will help future readers (***including yourself***) and will serve as documentation.

At the very least please describe the issue you are addressing, and what your fix entails. 

## Sample Card
https://user-images.githubusercontent.com/78950710/113285867-10285700-9309-11eb-9226-7f1f4af129a3.mp4

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp [hunk 1]
```diff
@@ -239,12 +237,9 @@ namespace RendererQml
                         context->addToShowCardLoaderComponentList(componentId, showCardAction);
 
                         //Add show card loader to the parent container
-                        if (addLoaderSeparator)
-                        {
-                            AddSeparator(uiContainer, std::make_shared<AdaptiveCards::Container>(), context);
-                            addLoaderSeparator = false;
-                        }
-                        auto uiLoader = std::make_shared<QmlTag>("Loader");
+                        AddSeparator(uiContainer, std::make_shared<AdaptiveCards::Container>(), context, true, uiAction->GetId() + "_loader");
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:240]: Can we not add the separator to the component instead?

**Review by rj100189 — CHANGES_REQUESTED** [review]

**Review by rj100189 — COMMENTED** [review]
Also, Please remove `#include <windows.h>` from [here](https://github.com/AdaptiveCards-Desktop/AdaptiveCards/pull/135/files#R4). This will break for Linux build.

**Comment by rj100189** [comment]
Also remove `#include <windows.h>` from `Utils.cpp`

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp (hunk 1 above)
Comment (jmenghan) [reply source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:240]: I have tried adding the separator in component in the outer most rect as well as inner rect (body layout) it then disturbs the background color of the card and also the top margin too
so went with this option instead

**Review by rj100189 — APPROVED** [review]

**Pull request merged**